oss-internship-2020/openjpeg/README.md

```diff
@@ -0,0 +1,8 @@
+# OpenJPEG
+
+This library provides sandboxed version of the [OpenJPEG](https://github.com/uclouvain/openjpeg) library. 
+
+
+## Examples
```

Comment:
Please add a section detailing how to build this and how to check out (i.e., mention `git submodule update --init --recursive`).